docker/frontend/files/usr/bin/init-database.sh

```diff
@@ -3,3 +3,8 @@
 cd /usr/share/copr/coprs_frontend/ && copr-frontend create-db --alembic alembic.ini
 copr-frontend create-chroot \
     $(ls /etc/mock/{fedora,epel}-*-{i386,x86_64}.cfg |xargs -I{} -n1 basename {} .cfg)
+
+# OIDC and Kerberos don't support auth on local developer instances without
+# workarounds. Let's have a default user for easier onboarding.
+copr-frontend add-user jdoe jdoe@example.com
+copr-frontend alter-user jdoe --admin
```

Comment:
we could get rid of it in the future by https://github.com/fedora-copr/copr/issues/3643

could you please link the issue there? I think I will implement it in the future if I'll see the todo too much often :D

Reply:
Good point, thank you. Updated.